CHANGELOG.md

```diff
@@ -6,4 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/ppom-validator/
+## [0.0.1]
+### Added
+- Adding PPOM middleware ([#5](https://github.com/MetaMask/ppom-validator/pull/5))
+- Adding PPOM controller ([#4](https://github.com/MetaMask/ppom-validator/pull/4))
+- Adding PPOM wasm code ([#3](https://github.com/MetaMask/ppom-validator/pull/3))
+- Add storage class for PPOM data ([#1](https://github.com/MetaMask/ppom-validator/pull/1))
+- Initialising the repo from https://github.com/MetaMask/metamask-module-template ([#2](https://github.com/MetaMask/ppom-validator/pull/2))
+
+### Changed
+- Restrict provider access to PPOM ([#7](https://github.com/MetaMask/ppom-validator/pull/7))
+- Integrating with ppom npm module ([#8](https://github.com/MetaMask/ppom-validator/pull/8))
```

Comment:
Nit: Maybe better to use imperative statements consistently rather than a mix
```suggestion
- Add PPOM middleware ([#5](https://github.com/MetaMask/ppom-validator/pull/5))
- Add PPOM controller ([#4](https://github.com/MetaMask/ppom-validator/pull/4))
- Add PPOM wasm code ([#3](https://github.com/MetaMask/ppom-validator/pull/3))
- Add storage class for PPOM data ([#1](https://github.com/MetaMask/ppom-validator/pull/1))
- Initialize the repo from https://github.com/MetaMask/metamask-module-template ([#2](https://github.com/MetaMask/ppom-validator/pull/2))

### Changed
- Restrict provider access to PPOM ([#7](https://github.com/MetaMask/ppom-validator/pull/7))
- Integrate with ppom npm module ([#8](https://github.com/MetaMask/ppom-validator/pull/8))
```